actionview/test/template/tag_helper_test.rb

```diff
@@ -176,11 +176,19 @@ def test_tag_with_dangerous_unknown_attribute_name
 
   def test_tag_builder_with_dangerous_unknown_attribute_name
     escaped_dangerous_chars = "_" * COMMON_DANGEROUS_CHARS.size
-    assert_equal "<the-name #{escaped_dangerous_chars}=\"the value\"></the-name>",
-                 tag.public_send(:"the-name", COMMON_DANGEROUS_CHARS => "the value")
-
-    assert_equal "<the-name #{COMMON_DANGEROUS_CHARS}=\"the value\"></the-name>",
-                 tag.public_send(:"the-name", COMMON_DANGEROUS_CHARS => "the value", escape: false)
+    if RUBY_VERSION >= "2.7"
+      assert_equal "<the-name #{escaped_dangerous_chars}=\"the value\"></the-name>",
+        tag.public_send(:"the-name", COMMON_DANGEROUS_CHARS => "the value")
```

Comment:
Have you tried:
 
```suggestion
        tag.public_send(:"the-name", { COMMON_DANGEROUS_CHARS => "the value" })
```

Pretty sure that will work across versions.

Reply:
That still fails:

```ruby
def tag_string(name, content = nil, **options, &block)
  puts "#{name}, #{content}, #{options.inspect}"
end

tag_string('a-name', "abc" => "def")
# a-name, {"abc"=>"def"}, {}
tag_string('a-name', {"abc" => "def"})
# a-name, {"abc"=>"def"}, {}
tag_string('a-name', :abc  => "def")
# a-name, , {:abc=>"def"}
```

I've changed the PR to call `to_sym` on the key instead.